src/info/langs/language.rs

```diff
@@ -2,14 +2,38 @@ use colored::Color;
 use serde::Serialize;
 use std::env;
 use strum::{EnumIter, EnumString, IntoStaticStr};
+use OptionalColor::*;
 
 pub struct Colors {
-    basic_colors: Vec<Color>,
-    true_colors: Option<Vec<Color>>,
+    basic_colors: Vec<OptionalColor>,
+    true_colors: Option<Vec<OptionalColor>>,
 }
 
+macro_rules! define_optional_color {
+    ($( $color:ident ),*) => {
+        #[derive(Debug)]
+        pub enum OptionalColor {
```

Comment:
I'm not sur about the name, what about `AsciiColor` instead

Reply:
Yeah, this was back when there was a `None` variant. `AsciiColor` sounds good to me.